EHR_SM/resources/queries/samples/AllSampleTypes.query.xml

```diff
@@ -0,0 +1,10 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="AllSampleTypes" tableDbType="TABLE" useColumnOrder="true">
+                <insertUrl></insertUrl>
+                <javaCustomizer class="org.labkey.ehr_sm.EHR_SMCustomizer  " />
```

Comment:
Not sure if it matters, but there is a space at the end of EHR_SMCustomizer

Reply:
Oops, thanks for catching. Fixed